cmd/mount.go

```diff
@@ -419,6 +421,12 @@ var mountCmd = &cobra.Command{
 			return Destroy(fmt.Sprintf("failed to initialize new pipeline [%s]", err.Error()))
 		}
 
+		// Dry run ends here
+		if options.DryRun {
+			log.Crit("Dry-run complete")
```

Comment:
Why is this logging as critical, instead of as a log.trace? It seems like this could confuse a user looking at the log file.

Reply:
I had trouble getting this message to show up in the log. But honestly, Crit didn't actually help. Something's weird about trying to log something when you're just about to exit.
Anyway, I'll switch this to trace.